index.html

```diff
@@ -4,8 +4,109 @@
+      <h2 class="heroSection_title">Handcrafted, home-made masterpieces</h2>
+
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <div class="form-container">
+          <input type="email" id="email" placeholder="Type your email" name="email" />
+          <input type="submit" id="submit" />
+        </div>
+      </form>
+
+      <div id="features">
+        <p class="firstParagraph"><h3>Premium Materials</h3>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+        <p class="firstParagraph"><h3>Fast Shipping</h3>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        <p class="firstParagraph"><h3>Quality Assurance</h3>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+      </div>
+
+      <section id="how-it-works">
+        <iframe id="video" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      </section>
+
+      <section id="pricing">
+        <div class="pricing_plan">
+          <div class="pricing_plan_head">
+            TENOR TROMBONE
+          </div>
+<!--test-->
+          <div class="pricing_plan_price">$500</div>
+
+          <div class="pricing_plan_body">
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+          </div>
+
+          <button class="pricing_button">SELECT</button>
+        </div>
+
+        <div class="pricing_plan">
+          <div class="pricing_plan_head">
+            BASS TROMBONE
+          </div>
+
+          <div class="pricing_plan_price">$500</div>
+
+          <div class="pricing_plan_body">
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+          </div>
+
+          <button class="pricing_button">SELECT</button>
+        </div>
+
+        <div class="pricing_plan">
+          <div class="pricing_plan_head">
+            VALVE TROMBONE
+          </div>
+
+          <div class="pricing_plan_price">$500</div>
+
+          <div class="pricing_plan_body">
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+          </div>
+
+          <button class="pricing_button">SELECT</button>
+        </div>
+      </section>
+
+      <a href="https://youtu.be/y8Yv4pnO7qc" id="video-link"></a>
+
+      <footer id="footer">
+        <p><a href="#">Privacy</a>
+            <a href="#">Terms</a>
+            <a href="#">Contact</a>
+        </p>
+        <p>Copyright 2016, Original Trombones</p>
+      </footer>
```

Comment:
This is expected to be a list item li and should be under an unordered list ul
Please change that and edit the CSS to remove the underline in links

![image](https://user-images.githubusercontent.com/43151556/155009284-ee8cebcf-c594-4bc3-8a45-8dffa5719d17.png)